portfolio/src/main/java/com/google/sps/servlets/CommentsHandler.java

```diff
@@ -12,6 +12,7 @@
 import com.google.gson.Gson;
 import java.lang.reflect.Field;
 import java.util.ArrayList;
+import java.util.List;
```

Comment:
Is this used?

Reply:
oh, seems like my IDE unused imports detection is broken, sure it is not used here